adagrams/game.py

```diff
@@ -1,11 +1,148 @@
+import random
+
+def create_letter_pool():
```

Comment:
I love this approach of using a helper function to manage creating a pool of letters with the correct frequencies!